src/SFA.DAS.EmployerIncentives.Web/Views/Payments/ListPayments.cshtml

```diff
@@ -142,7 +142,7 @@
                             <partial name="_PaymentStatus" for="@apprenticeApplication.FirstPaymentStatus" />
                         </td>
                         <td data-label="Second payment" class="govuk-table__cell">
-                            @if (!apprenticeApplication.FirstPaymentStatus.ShowPaymentStatus)
+                            @if (apprenticeApplication.FirstPaymentStatus != null && !apprenticeApplication.FirstPaymentStatus.ShowPaymentStatus)
```

Comment:
Be better if this was a property on the model ad the logic applied there. e.g. Model.ShowSecondPaymentStatus